src/placeos-models/asset.cr

```diff
@@ -0,0 +1,53 @@
+require "./base/model"
+
+module PlaceOS::Model
+  class Asset < ModelBase
+    include RethinkORM::Timestamps
+
+    table :asset
+
+    attribute name : String, es_subfield: "keyword"
+    attribute category : String = ""
+    attribute description : String = ""
+    attribute size : String = ""
+
+    attribute purchase_date : Int32
+    attribute good_until_date : Int32?
+
+    attribute barcode : Int32
+    attribute brand : String = ""
+
+    attribute purchase_price : Int32
+
+    # Array of URLs to images for an asset
+    attribute images : Array(String) = ->{ [] of String }
+
+    # URL of downloadable receipt
+    attribute invoice : String?
+
+    attribute quantity : Int32 = 0
+    attribute in_use : Int32 = 0
+
+    attribute other_data : JSON::Any = JSON::Any.new({} of String => JSON::Any), converter: JSON::Any::StringConverter, es_type: "text"
```

Comment:
Any justifcation for this converter?

Reply:
`es_type: "object"` may be more appropriate here, then the data in `other_data` will be indexed like an object